apps/address-book/.eslintrc.js

```diff
@@ -57,6 +57,31 @@ module.exports = {
     'react/no-unused-prop-types': 'warn',
     'sort-imports': ['warn', { ignoreDeclarationSort: true }],
   },
+  "overrides": [
+    {
+      "files": ["test/*.test.js"],
+      "env": {
+        "mocha": true,
+      },
+      "globals": {
+        "contract": "readonly",
+        "artifacts": "readonly",
+        "web3": "readonly",
+        "assert": "readonly",
+        "expect": "readonly",
+      },
+      rules: {
+        "import/no-unused-modules": [
+          "off",
+          {
+            unusedExports: true,
+            missingExports: true,
+            ignoreExports: [],
+          }
+        ],
+      }
+    }
+  ],
```

Comment:
Do we want these changes as part of this PR, or is this something we should add across all apps in a separate PR?

Reply:
We did the same for other apps independently, but we can definitely isolate these into a new PR and it makes sense for me as well